### ISSUE TITLE
Original naming of characters an other things...

### ISSUE THREAD

**Issue by Bertram25** [solarus-games/children-of-solarus#2]
@christopho 

This issue is about the work toward renaming names, files, and story bits to make them fully original.

For instance, the repository name needs to get an original name:
Is 'Children of Solarus' ok? (I like this one :])

Shall the princess be called Zenna? (Princess Solaritine? See below)

Let's put here the questions that pops up along that WIP.

**Comment by Diarandor** [comment]
I will give my opinion:

1) Since we will be using the art of the hero I created (I called him Edgar), would you like to keep the original name I chose? We are not forced to use that name anyway, so it is not necessary. But I would like to use it since I got used to call him Edgar (otherwise maybe we should rename the pixel-art files, because they appear refering to "Edgar" in the filenames). "Edgar" is a rare name, which is good to make our character more unique. For the last name, what about "Tunder"? (http://www.surnamedb.com/Surname/tunder). Other variations are "Thunder", "Tander" or "Tender".
Since the sword animation we have has some electricity animation on the sword, I thought that something related to thunders or wind could be fine for the last name. (The superspin attack, when drawn, will have both electric and wind animations, so our hero could be related to the electric/wind elements somehow in the story.)

2) IMHO, "Children of Solarus" sounds a little bit childish (no offense intended). You will probably disagree, but I propose to use something like "Descendants of Solarus", or something similar, which sounds more mature for the name of the game.

3) To substitue the princess sprites, we could probably use the character "Robyne Alabaster" I drew a lot of time ago (it will be redrawn and improved someday). If that's the case, what do you think of keeping the name "Robyne" for the name of the princess? (Well... if you don't like it, there is no problem in using a different one.) The last name "Alabaster" means crossbow, which is cool, and it is a very rare one (http://www.surnamedb.com/Surname/alabaster). Her current sprite image can be found here:
http://diarandor.deviantart.com/art/Robyne-v1-0-589305384?q=gallery%3ADiarandor%2F57740730&qo=95
and here:
http://diarandor.deviantart.com/art/Dialog-Robyne-v1-0-589309243?q=gallery%3ADiarandor%2F57740730&qo=93

**Comment by Diarandor** [comment]
To replace the triforce, an option is to use the ancient symbol known as "Triskelion" or "Triskele":
https://en.wikipedia.org/wiki/Triskelion
It is similar to a triangle, but since the symbol exists there will be no problem with Nintendo. It consists of a triple spiral and there are several variants of the symbol. It is a Celtic and pre-Celtic symbol. More links here:
https://thecelticjourney.wordpress.com/tag/triskelion/
https://en.wikipedia.org/wiki/Triple_spiral
In the last link you can read:

> The triple spiral is one of the main symbols of Celtic Reconstructionist Paganism, often standing for the "three realms" - Land, Sea and Sky, or for one of a number of deities who are described in the lore as "threefold" or triadic.[1] The god Manannán is probably most often the one symbolized by the triskele, though some also use it for the goddess Brighid. Some Celtic-inspired Wiccans also use the triple spiral symbol, most often to represent the concept of the triple goddess. <

Therefore, we could use a triple goddess instead of the three goddesses, if we use the true history of that symbol, which is very interesting. It has several meanings, one of them is "Land, Sea and Sky". It also symbolizes "life, death and afterlife", and has other interpretations. We can also create a new meaning if you prefer (like the elements "fire/lava, "water/ice", "electricity/wind"), etc.

**Comment by MCMic** [comment]
For me Edgar is the character from the game «Legend of Edgar».

**Comment by Diarandor** [comment]
I never heard of that game before. Well... there is also an Edgar in FFVI. I never said it was a unique name, I said it is a rare name, and I like it.

**Comment by Renkineko** [comment]
I don't have any opinion about the names, but I like the idea of the triskele and the triple goddess. Or god.

**Comment by christopho** [comment]
We don't need a replacement for the Triforce, it almost does not appear in ZSDX.

**Comment by oclero** [comment]
I don't like the idea of changing the title, I spent too much time on creating the logo :-P

**Comment by christopho** [comment]
I also like "Children of Solarus". It sounds like "Children of Dune" :)

**Comment by Diarandor** [comment]
Ok, then forget about changing the name of the game. :)

**Comment by Diarandor** [comment]
I was wrong about Edgar being a rare name. We can still use it or use the one you chose before (Lunarius). Or use both of them, where one of them would be the last name: Edgar Lunarius or Lunarius Edgar.

As for the zoras, I propose to call them fishmen (or something similar).

**Comment by Zefk** [comment]
Edgar does not have to be completely thrown out. With a simply twist it becomes unique and rare.
**Examples:**
Edzager Lunarius
Edgarger Lunarius
Edgarter Lunarius
Ediarxor Lunarius
Edvoth Lunarius Fargernoth 

From reading above....I kinda like the sound of, "Descendants of Solarus." It gives it a more mature and mysterious feel about it.

**Comment by christopho** [comment]
I like both Children of Solarus and Descendants of Solarus!

**Comment by oclero** [comment]
Why do you want to change the name of the hero ? The player can set his own name, and does not even need to know the "real" name of the character.

**Comment by Zefk** [comment]
I think he would still need a name. Link has a name and I always typed that name while playing Zelda. Normally there are names to pick and an option input a name not on the list. At least from the games I have played. Pokemon, Sword of Mana, etc

**Comment by Diarandor** [comment]
@oclero: Yes, you are right, we should allow the player to choose the name of the character. But still there should be a default name. The name of the hero could be important for us since it may become representative/distinctive of the Solarus community since his graphics will be distributed with the Solarus sample quest. (However I am not the one who corresponds to take that decision, I am just giving my opinion.)

**Comment by Zefk** [comment]
@christopho An idea hit me. Descendants of Solarus could be the sequel to Children of Solarus.

**Comment by christopho** [comment]
Here are a few suggestions for the hero sprite name instead of Edgar, after discussing with Mymy:
Eldran
Eldrick
Edvald
Elyas
Endrall

What do you think? It will be the hero name for the Solarus default quest resource pack.

Here is a small poll: http://www.strawpoll.me/10823217

@Diarandor @Bertram25 @oclero

**Comment by christopho** [comment]
And a suggestion from Mymy for Zelda:
Princess Solana
To me it is better than Princess Solarina.

**Comment by Diarandor** [comment]
For the hero, I like both Eldrick and Elyas.

For the princess, I don't like Solarina either, it sounds strange to me; Solana seems better to me. Other options could be: Solange, Solene or Sollemnia. http://www.behindthename.com/name/solange
(Anyway, her name does not need to begin with "Sol" of "Solarus". My previous choice for her name was Robyne, which is another option if you don't dislike it.)

**Comment by Nate-Devv** [comment]
You're right, Princess Solana sounds much more regal.

**Comment by Diarandor** [comment]
This issue is very old, and some decissions have been taken a lot of time ago, like these names:
Hero: Link -> Eldran
Princess: Zelda -> Robyne
Elder (wise man and wizard): Sahasrahla -> Kaleido

The most important name that remains to choose is the name of the bad guy.
